public/js/script.js

```diff
@@ -189,22 +207,32 @@ function saveRecordedStream(stream, user) {
 
     let file = new File([blob], `${user}-${moment().unix()}-record.webm`);
 
-    // let fd = new FormData();
-    // fd.append("fileName", file);
-    // fd.append("data", blob);
-    // console.log(fd)
-    // $.ajax({
-    //         type: 'POST',
-    //         url: '/',
-    //         data: fd,
-    //         processData: false,
-    //         contentType: false
-    //     }).done(function(data) {
-    //         console.log(data)
-    //     })
-    saveAs(file);
-}
+    // create FormData
+    var formData = new FormData();
+    formData.append('videoFilename', file.name);
+    formData.append('videoData', file);
+
+    for (var key of formData.entries()) {
```

Comment:
This seems like, a useless loop? For testing maybe?

Reply:
Yes, removed all unwanted code and comments